server/index.js

```diff
@@ -0,0 +1,21 @@
+require('dotenv/config')
+const path = require('path')
+const express = require('express')
+const mongoose = require('mongoose')
+const bodyParser = require('body-parser')
+const feedRouter = require('./feed-router')
+const { PORT, MONGO_DB_URI } = process.env
+
+const app = express()
+mongoose.connect(MONGO_DB_URI)
```

Comment:
If you are not using mongoose yet, you should delete all of this. It's important not to check in extra code.